test/create.test.ts

```diff
@@ -221,26 +278,21 @@ CREATE TABLE cooltable (invalid)
     ])
       .command(mod)
       .parse();
-    assert.calledWith(
-      consoleLog,
-      match(function (value: any) {
-        value = JSON.parse(value);
-        const { prefix, name, chainId, tableId, transactionHash } =
-          value.meta.txn;
-        return (
-          prefix === "file_test" &&
-          chainId === 31337 &&
-          name.startsWith(prefix) &&
-          name.endsWith(tableId) &&
-          typeof transactionHash === "string" &&
-          transactionHash.startsWith("0x")
-        );
-      }, "does not match")
-    );
+
+    const res = consoleLog.getCall(0).firstArg;
+    const value = JSON.parse(res);
+    const { prefix, name, chainId, tableId, transactionHash } = value.meta.txn;
+
+    equal(prefix, "file_test");
```

Comment:
I was having trouble with the usage of `assert.calledWith(` because no matter what went wrong with the test the error message was "does not match" and there was no line number.  In some cases console.log had been called twice making things even harder to track down.
I switched to only looking at the first call and using separate assertions for each property.  If this seems like a good pattern, we could make a separate PR with changes to the rest of the tests.

Reply:
I'm good with this, it's easier to read.